base/shell/explorer/explorer.cpp

```diff
@@ -49,6 +49,41 @@ static VOID InitializeAtlModule(HINSTANCE hInstance, BOOL bInitialize)
     }
 }
 
+static BOOL
+GetServerAdminUIDefault()
+{
+    BOOL server;
+    DWORD value = 0, size = sizeof(value);
+    LPCWSTR rosregpath = L"SYSTEM\\CurrentControlSet\\Control\\ReactOS\\Settings\\Version";
+    if (SHGetValueW(HKEY_LOCAL_MACHINE, rosregpath, L"ReportAsWorkstation", NULL, &value, &size) == NO_ERROR)
+        server = value == 0;
+    else
+        server = IsOS(OS_ANYSERVER);
+    return server && IsUserAnAdmin();
+}
+
+static void
+InitializeServerAdminUI()
+{
+    HKEY hKey = SHGetShellKey(SHKEY_Root_HKCU | SHKEY_Key_Explorer, L"Advanced", TRUE);
+    if (hKey)
+    {
```

Comment:
```suggestion
    if (!hKey)
        return;
```
and you would gain one level of indentation for the rest of the code below.